### ISSUE TITLE
Feature Request: Total Balance over Time (in native currency) - Line Chart and Table Column

### ISSUE THREAD

**Issue by pax85** [digitalbitbox/bitbox-wallet-app#1709]
Hi,

I'd love to be able to know my total balance in the native account currency (BTC, ETH) after every transaction and as a chart:

### Balance Column
The transactions table should have an additional column with total overall balance per coin after every transaction. 

e.g. Krakens History Page: https://support.kraken.com/hc/en-us/articles/360024871131-How-to-find-your-balances-on-a-particular-date

### Balance Line Chart
On the per account page (e.g. BTC or ETH, ...) you could add your line chart (like from the portfolio page) and display the total balance over time in either the native currency (BTC, ETH) or in Fiat Value (switchable). 

--
kind thanks for this product btw.

**Comment by jadzeidan** [comment]
Hi @pax85, thanks for the request. I think it would be possible to add such an option to display the transaction history in that "bank statement" style or add a way to export it in that format (maybe being able to select a date range as well). I would probably not show it in that style by default because it is quite technical looking. But as a "power user" option it could work.

Regarding the balance chart, yes I envision that we could add a chart to each account separately as well. We could also keep the functionality of being able to toggle the units (by clicking on it) which would allow you to display the chart in the units of the coin you are using, so you can see how much you are accumulating over time.

**Comment by benma** [comment]
The second request is a duplicate of https://github.com/digitalbitbox/bitbox-wallet-app/issues/1632 I think.

**Comment by pax85** [comment]
> The second request is a duplicate of #1632 I think.

Yes, seems like it. With the addition of showing the chart in the native currency :)
